bigquery_etl/shredder/bug1751979.py

```diff
@@ -0,0 +1,251 @@
+#!/usr/bin/env python3
+
+"""Constants for use in sanitizing main_v4 for bug 1751979."""
+
+BUG_1751979_MAIN_V4_UDFS = """
+CREATE TEMP FUNCTION sanitize_search_counts(input ANY TYPE) AS (
+  (
+    WITH base AS (
+      SELECT
+        key,
+        value,
+        REGEXP_EXTRACT(key, "([^.]+\\.in-content[:.][^:]+:).*") AS prefix,
+        REGEXP_EXTRACT(key, "[^.]+\\.in-content[:.][^:]+:(.*)") AS code,
```

Comment:
there's actually two layers of escaping needed here (python, then bq), so it would be `\\\\.`, but instead better to just use `[.]`:
```suggestion
        REGEXP_EXTRACT(key, "([^.]+[.]in-content[:.][^:]+:).*") AS prefix,
        REGEXP_EXTRACT(key, "[^.]+[.]in-content[:.][^:]+:(.*)") AS code,
```

Reply:
I think it would be good to make this whole string raw with an `r` prefix too.